stdlib/3/_importlib_modulespec.pyi

```diff
@@ -9,6 +9,8 @@ from abc import ABCMeta
 import sys
 from typing import Any, Optional
 
+MYPY_MODULE = 'importlib.machinery'
```

Comment:
If we go down this route, we should prefix it with `_` to make clear that this name isn't actually part of this module. However, I'd prefer to just hardcode the location of ModuleType in mypy and not add special handling in typeshed.